src/components/Analisis.tsx

```diff
@@ -27,17 +45,17 @@ export default function Analisis({ min, max, total, tipos, componentes }) {
       </div>
 
       <div className={styles.analisisDatos}>
-        {Object.keys(tipos.byName).map((t, i) => (
+        {Object.entries(tipos.byName).map(([t, value], i) => (
```

Comment:
this is a functional change and should go to it's own commit.
as a rule you should refrain to 'better the code' while migrating.

Reply:
I'll keep it in mind in the future.

You see, when you access `tipos.byName[t]` you don't have a guarantee that that doesn't evaluate to `undefined` in Typescript. That's because `byName` isn't a `Record<NarrowType, Whatever>` but a `Record<string, string>`, which makes it possible to access `byName["someString"]` and get `undefined`.

In my opinion this change is pertinent to the commit.

Reply:
then the right way is to put it in another commit explaining exactly what you wrote here.